fluent-bit/deb/debian/launch-fluent-bit.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020, Arm Limited and affiliates.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------
+
+IDENTITY_JSON=${IDENTITY_JSON:-/var/lib/pelion/edge_gw_config/identity.json}
+DEVICE_ID=$(jq -r .deviceID ${IDENTITY_JSON})
+if [ $? -ne 0 ]; then
+    echo "Unable to extract device ID from identity.json"
+    exit 1
+fi
```

Comment:
Do we need identity.json for fluentbit?

Reply:
We don't need this in distro, but we do in snap.
Because systemd is watching edge-proxy, we do not need to check for identity. That was only cause snap wasn't able to wait. I will remove